CONTRIBUTING.md

```diff
@@ -40,7 +40,42 @@ approach.
 
 Please include tests in your pull request.
 
-We recommend making your pull request from a fork. See [Creating a pull
+We recommend making your pull request from a fork. See [creating a pull
 request from a
 fork](https://help.github.com/articles/creating-a-pull-request-from-a-fork/)
 for more information.
+
+## Stability
+
+It’s important that our users can depend on our product, and not to worry
+about changes in model-viewer causing regressions in their use of the
+component. However, it’s also important that we continue to improve
+model-viewer, making changes to improve ergonomics and rendering quality.
+
+In general, <model-viewer> follows semver. Major versions (`X.y.z`) are used for
+the largest features and backwards-incompatible changes (more below on that),
+minor versions (`x.Y.z`) are used for new features, and patch versions (`x.y.Z`)
+are used for bug fixes.
+
+When possible please keep your changes backwards-compatible - including
+keeping both existing APIs and rendering consistent. When that’s not possible,
+we’ll follow a three-step process across several [semver](semver.org) major
+versions (in the `0.y.z` versions leading up to 1.0, we’ll use the minor
+version - `y` - for the same purpose): 
+
+1) Add a new optional API to enable the new behavior; in the release notes,
+document the new behavior, as well that it will become the default in the
+next version (in for instance the `2.0.0` release)
+1) Make the new API default enabled (allow it to be explicitly disabled for
+the old behavior); in the release notes, document this change, and that the
+API will be removed in the next version (the `3.0.0` release)
+1) Remove the API (the `4.0.0` release)
```

Comment:
If we say the API will be removed in the 3.0.0 release, then what is 4.0.0 for?

Reply:
Actually removing it - in 3.0.0 we document that it will be removed, and then it's not removed until 4.0.0.

Reply:
I think I'm having some trouble groking whether the named version refers to the step you're writing about or the step after. I'm pretty sure one way or another there are only two named versions involved. Adding doesn't break, so doesn't increment. Changing the default and removing the old behavior both do increment. Something in this wording isn't consistent with that.